microshift_updating/microshift-update-rpms-manually.adoc

```diff
@@ -6,7 +6,7 @@ include::_attributes/attributes-microshift.adoc[]
 
 toc::[]
 
-Updating {product-title} for non-OSTree systems such as {op-system-base-full} requires downloading then updating the RPMs. For patch releases, such as 4.16.1 to 4.16.2, download and update the RPMs. For minor-version release updates, add the step of enabling the update repository using your subscription manager.
+Updating {product-title} for non-OSTree systems such as {op-system-base-full} requires downloading then updating the RPMs. For patch releases, such as 4.17.1 to 4.17.2, download and update the RPMs. For minor-version release updates, add the step of enabling the update repository using your subscription manager.
 
 [IMPORTANT]
 ====
```

Comment:
There are conflicting statements in the "Manual updates with RPMs" section. Both are in IMPORTANT boxes.
The first and correct text is:
"MicroShift updates are supported with a maximum of two minor versions. For example, you can update to 4.16 from 4.14 in a single step. Updating 4.14 to 4.15 first is not required."

The second farther down on the same page says the contrary:
"You can only update MicroShift from one version to the next in sequence. Jumping minor versions is not supported. For example, must update 4.15 to 4.16."

Reply:
@DanielFroehlich should the important section in the link https://78509--ocpdocs-pr.netlify.app/microshift/latest/microshift_updating/microshift-update-rpms-manually#microshift-updating-rpms_microshift-update-rpms-manually be modified or removed?

Reply:
good catch @jogeo !
@aravipra , Yes, please remove the second father down Important box. That is a left over from the early days.

Reply:
done